resolve/matrix/Sparse.hpp

```diff
@@ -6,6 +6,13 @@
 #include <resolve/MemoryUtils.hpp>
 
 namespace ReSolve { namespace matrix {
+  /**
+   * @brief This class implements basic sparse matrix interface. (Almost) all sparse matrix formats store information about matrix rows and columns (as integers) and data (as real numbers).
+   *        This class is virtualand implements only what is common for all basic formats.
+   *        Note that regardless of how row/column information is stored, all values need to be stored, so all utilities needed for values are implemented in this class.
+   *
+   * @author Kasia Swirydowicz <kasia.swirydowicz@pnnl.gov>
+   */
```

Comment:
Split brief from description:
```suggestion
  /**
   * @brief This class implements basic sparse matrix interface. 
   *
   * (Almost) all sparse matrix formats store information about matrix rows and columns (as integers) 
   * and data (as real numbers).
   * This class is virtualand implements only what is common for all basic formats.
   * Note that regardless of how row/column information is stored, all values need to be stored, 
   * so all utilities needed for values are implemented in this class.
   *
   * @author Kasia Swirydowicz <kasia.swirydowicz@pnnl.gov>
   */
```